code/game/objects/effects/decals/Cleanable/humans.dm

```diff
@@ -21,22 +22,28 @@ GLOBAL_LIST_EMPTY(splatter_cache)
 	var/amount = 5
 	var/dry_timer = 0
 	var/off_floor = FALSE
-
+	var/image/weightless_image
+	inertia_move_delay = 1 // so they dont collide with who emitted them
 
 /obj/effect/decal/cleanable/blood/replace_decal(obj/effect/decal/cleanable/blood/C)
+	if(C == src)
+		return FALSE
 	if(C.blood_DNA)
 		blood_DNA |= C.blood_DNA.Copy()
 	if(bloodiness)
 		if(C.bloodiness < MAX_SHOE_BLOODINESS)
 			C.bloodiness += bloodiness
 	return ..()
 
-
 /obj/effect/decal/cleanable/blood/Initialize(mapload)
 	. = ..()
+	weightless_image = new()
```

Comment:
Do we actually need this line give that in `update_icon` we do `weightless_image = image(...)`?

Reply:
Yes since they won't get the zero g sprites otherwise.

Reply:
Alright, I see.